packages/icon/src/react/index.tsx

```diff
@@ -35,9 +22,20 @@ export interface IconComponent
   extends RefForwardingComponent<IconProps, HTMLDivElement, IconStatics> {}
 
 const Icon = React.forwardRef((props, ref) => {
-  const { size = sizes.medium, color, ...rest } = props
-
-  return <div {...style.icon({ color, size })} {...rest} ref={ref} />
+  const { className, size = sizes.medium, color, ...rest } = props
+
+  return (
+    <div
+      {...rest}
+      className={classNames(
+        'psds-icon',
+        `psds-icon--size-${size}`,
+        `psds-icon--color-${color}`,
```

Comment:
There is no default color on icon so we should ```color && `psds-icon--color-${color}` ``` I noticed classes with undefined in the icon snapshot.

Reply:
agreed. done.